canonical-websites/www.ubuntu.com
# Flattened /legal/ubuntu-advantage section

## Done

- Flattened legal ubuntu-advantage section
- Updated navigation
- Updated redirects
- Added some special redirects requested by legal team

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- look at the files in the /legal directory and compare to this [spreadsheet](https://docs.google.com/spreadsheets/d/12kuuGgwRqN3k7wlRc4QKIuZifjvxbynizO_LEjKjBq4/edit#gid=0)

* test the link [/legal/ubuntu-advantage-service-description](http://0.0.0.0:8001/legal/ubuntu-advantage-service-description) and [the old link](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description) redirects also test that [/uasd](http://0.0.0.0:8001/uasd) redirects as well
* test the link [/legal/ubuntu-advantage-service-description/ja](http://0.0.0.0:8001/legal/ubuntu-advantage-service-description/ja) and [the old link](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description-ja) redirects
* test the link [/legal/ubuntu-advantage-assurance](http://0.0.0.0:8001/legal/ubuntu-advantage-assurance) and [the old link](http://0.0.0.0:8001/legal/ubuntu-advantage/assurance) redirects
* test the link [/legal/ubuntu-advantage-service-terms](http://0.0.0.0:8001/legal/ubuntu-advantage-service-terms) and [the old link](http://0.0.0.0:8001/legal/ubuntu-advantage/service--terms) redirects

## Issue / Card

Fixes #4200
